test/Bookings-test.js

```diff
@@ -4,3 +4,48 @@ import User from '../src/User';
 import Bookings from '../src/Bookings';
 import Rooms from '../src/Rooms';
 import {customerData, bookingsData, roomData} from './test-data';
+
+describe('Booking', () => {
+  let booking1, booking2
+
+  beforeEach(() => {
+    booking1 = new Bookings(bookingsData[0]);
+    booking2 = new Bookings(bookingsData[1]);
+  });
+
+  it('Should be a function', () => {
+    expect(Bookings).to.be.a('function');
+  });
+
+  it('Should be an instance of booking', () => {
+    expect(booking1).to.be.an.instanceof(Bookings);
+  });
+
+  it('should have an id', () => {
+    expect(booking1.id).to.equal('5fwrgu4i7k55hl6sz');
```

Comment:
For values in tests like this, where you're expecting the result from the class to equal the value from the mock data, I personally prefer to reference the initial object, in case I change something in the mock data, I don't have to re-copy/paste throughout. So this could be something like `expect(booking1.id).to.equal(bookingsData[0].id);`

Works either way though!